content/en/core/process/workflow.md

```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
  - Remove the `Needs Triage` label once triage is complete
  - Document the reasoning by commenting in the issue. This will help reduce mistakes, as the reasoning will be available for everyone to read, and any mistakes there can be rectified.
  
-# Workflow for contributing to cht-docs
-Unlike the above development workflow, the workflow for contributing to [cht-docs](https://github.com/medic/cht-docs) does not involve release management and Acceptance Testing. In order to streamline the process of contributing to docs while still maintaining quality, please submit a pull request with any changes to [cht-docs](https://github.com/medic/cht-docs). One of the repository's maintainers will review the PR, request additional changes as needed, and merge the PR when it is ready. 
+# Documentation Workflow
 
-Where possible, please follow our [docs style guide]({{< ref "docs-style-guide" >}}).
+This documentation site (and related [GitHub site](https://github.com/medic/medic.github.io)) is the CHT’s main resource for informing users on the product. It is designed to provide Community Health Toolkit users clear information on product functionality, as well as the necessary guidelines to accomplish key steps for setting up, deploying, and maintaining the product. 
 
+Documentation is a collaborative effort among all Community members. To improve information on the documentation site, we encourage commenting, asking questions, and sharing ideas for improvement.
```

Comment:
It is great to call out all community members to collaborate, although it is not clear from this line how they would do it. Would be helpful here to mention that they could contribute edits too. 

Can you reword this to serve as a lead-in to the rest of the docs workflow sections?

Reply:
updated and included a "getting started" section